drivers/openstack/openstack.go

```diff
@@ -480,12 +523,25 @@ func (d *Driver) Create() error {
 			return err
 		}
 	}
+	if d.BootFromVolume == false && d.VolumeSize > 0 {
```

Comment:
Just to validate, this is for the "create a VM with a block volume, but not booting from it" case?

Reply:
yes, correct !